src/main/java/games/strategy/triplea/ui/menubar/LobbyMenu.java

```diff
@@ -423,4 +428,10 @@ private void addExitMenu(final JMenu parentMenu) {
       parentMenu.add(menuFileExit);
     }
   }
+
+  private static Date toDate(final TemporalAmount amount) {
+    return amount != null
```

Comment:
The caller of this private method should always know if the parameter is null. If you call this method, you would expect for it to do something, so calling it with a null parameter is very likely to be a bug. Would you mind moving the 'null' handling upstream to the caller @RoiEXLab ?

Reply:
I agree with @DanVanAtta that we should be avoiding `null` when possible.  Just because the database layer defines a `NULL` column value to represent forever doesn't mean the domain model has to use a `null` `TemporalAmount` to do the same.

However, I'm going to assume you're doing this because the `BanUsernameController` and `BanMacController` public APIs use a `null` `Date` to represent forever, and this method is simply acting as an adapter between your internal representation (`@Nullable TemporalAmount`) and the controller APIs (`@Nullable Date`).  Unless we undertake a larger scale refactoring, we're probably stuck with returning `@Nullable Date` from this method to conform to the controller APIs.  But, as suggested above, you might be able to avoid using `@Nullable TemporalAmount` if you use some kind of magic constant or equivalent alternative to represent forever within this class.

Regardless, if you do end up using nullable types in any of the code, would you mind annotating the types with `@Nullable` so that possibility is clear to a future maintainer?